bounties_api/std_bounties/seo_client.py

```diff
@@ -0,0 +1,56 @@
+from bounties import settings
+from bounties.sns_client import sns_publish
+from bounties.utils import base_url_for, bounty_url_for, profile_url_for
+from std_bounties.models import Bounty
+from user.models import User
+from uuid import uuid4
+
+
+class SEOClient:
+
+    def __init__(self):
+        pass
+
+    def publish_new_sitemap(self, platform):
+        if (platform == 'bounties-network' or platform == 'gitcoin') and settings.ENVIRONMENT == 'rinkeby':
+            return
+        if platform not in PLATFORM_MAPPING and platform != 'gitcoin':
+            return
+        url = base_url_for(platform)
+        domain = url.replace('https://', '')
+        base_api_url = 'https://rinkebystaging.api.bounties.network/'
+        if settings.ENVIRONMENT == 'rinkeby':
+            base_api_url = 'https://rinkebystaging.api.bounties.network/'
+        sitemap_url = '{}sitemap.xml?domain={}'.format(base_api_url, domain)
+        if url != 'https://explorer.bounties.network':
+            sitemap_url = '{}&platform__in={}'.format(sitemap_url, platform)
+
+        sns_publish('sitemap', {'url': sitemap_url, 'bucket': domain})
+        sns_publish('ssrcache', {'url': url + '/explorer'})
+        sns_publish('ssrcache', {'url': url + '/'})
+        sns_publish('ssrcache', {'url': url})
+
+    def clear_cache(self, platform, bounty_id):
+        base_url = base_url_for(platform)
+        bounty_url = bounty_url_for(bounty_id, platform)
+        sns_publish('ssrcache', {'url': bounty_url})
+
+    def bounty_preview_screenshot(self, platform, bounty_id):
+        bounty = Bounty.objects.get(bounty_id=bounty_id)
```

Comment:
Could this ever be triggered by something external? Would it crash if the `bounty_id` doesn't exist?